buf.yaml

```diff
@@ -14,5 +14,7 @@ lint:
       - openfga/v1/errors_ignore.proto
     ENUM_VALUE_PREFIX:
       - openfga/v1/errors_ignore.proto
+      - openfga/v1/openfga_service.proto
     ENUM_ZERO_VALUE_SUFFIX:
-      - openfga/v1/errors_ignore.proto
\ No newline at end of file
+      - openfga/v1/errors_ignore.proto
+      - openfga/v1/openfga_service.proto
```

Comment:
This is needed to stop lint from failing on the non-standard consistency preference values. Previously we had lint ignore comments in the source, but there is no way to prevent those from getting into the generated swagger :(

Reply:
Moved the enum to its own proto file so we can just ignore that one file without needing to ignore rules for all of openfga_service.proto. Build failing on the breaking changes check, I don't think it's accounting for the definition just having been moved and not removed entirely.

Reply:
Updating my OpenFGA to use `/api/proto` of the latest commit (`8af050e60cbe201f3e20c48b0268d978c9f8105a`), things are working as normal and `ConsistencyPreference` is there and unchanged, so I don't think the build failure is correct in this case. @rhamzeh has observed the same.